frontend/src/features/leases/detail/LeasePages/deposits/components/DepositsReceivedContainer/columns.tsx

```diff
@@ -29,14 +29,21 @@ export class DepositListEntry {
     this.depositDescription = baseDeposit.description;
     this.amountPaid = baseDeposit.amountPaid;
     this.paidDate = baseDeposit.depositDate || '';
-    this.personDepositHolder = baseDeposit.personDepositHolder;
-    this.organizationDepositHolder = baseDeposit.organizationDepositHolder;
+    this.contactHolder = baseDeposit.contactHolder;
     this.depositReturnCount = baseDeposit.depositReturns.length;
   }
 }
 
 function renderHolder({ row: { original } }: CellProps<DepositListEntry, string>) {
-  return original.personDepositHolder?.fullName || '';
+  if (original.contactHolder) {
+    if (original.contactHolder.person) {
+      return original.contactHolder.person.firstName + ' ' + original.contactHolder.person.surname;
```

Comment:
Do we have requirements on middle name here?

Reply:
also this will result in an extra space if the surname is null/empty/undefined.

Reply:
There is a utility in personUtils for contacts that we could maybe reuse here.

Reply:
Given that we were talking about how to display names I am not sure what is the standard. I believe we will standardize the person name everywhere in a later story

Reply:
I'm not aware of such a story. Something like this isn't really appropriate for tech debt either since it affects requirements. Maybe check in with the BAs?

Reply:
I double-checked and it states middle name too, I've added it